src/index.js

```diff
@@ -98,6 +102,9 @@ export default function prepass(
       c.__v = vnode;
       c.props = props;
       c.context = context;
+      if (c.state === undefined) {
```

Comment:
Not related to your `avoid-state` PR but I stumbled upon the fact that this was missing